flang/runtime/internal-unit.h

```diff
@@ -48,6 +55,10 @@ template <Direction DIR> class InternalDescriptorUnit : public ConnectionState {
   void BlankFillOutputRecord();
 
   StaticDescriptor<maxRank, true /*addendum*/> staticDescriptor_;
+  RT_OFFLOAD_VAR_GROUP_BEGIN
+  static constexpr std::size_t ownBufferSizeInBytes{1024};
+  RT_OFFLOAD_VAR_GROUP_END
+  bool usesOwnBuffer{false};
```

Comment:
The naming convention in the runtime (and front-end) has an underscore at the end of the name of a private `class` data member.